test/Dialect/FIRRTL/extract-instances.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: circt-opt --firrtl-extract-instances --verify-diagnostics %s | FileCheck %s
+// RUN: circt-opt --firrtl-extract-instances %s | FileCheck %s
```

Comment:
I guess this wasn't actually being used?

Reply:
Yeah, it wasn't used. I moved it to its own commit.